### DISCUSSION TITLE
Display EU average emissions

### DISCUSSION THREAD

**Discussion by maingoh** [electricitymaps/electricitymaps-contrib#2622] [General]
It would be nice to display the EU average emissions and maybe other big zones (USA, south america, africa ..) for comparison

**Comment by Kongkille** [comment]
I agree. We are currently working on a "historical" view, which will display an aggregate value for the zone. Not sure how we would be able to include larger zones, but it's definitely an interesting idea. Perhaps OOS for the live map, but perhaps if done right.
